example7/spdx/example7-third-party-modules.spdx.json

```
@@ -17,31 +17,31 @@
   ],
   "packages": [
     {
-      "packageName": "golang.org/x/text",
-      "SPDXID": "SPDXRef-go-module-golang.org/x/text",
-      "downloadLocation": "go://golang.org/x/text@v0.0.0-20170915032832-14c0d48ead0c",
+      "name": "golang.org/x/text",
+      "SPDXID": "SPDXRef-go-module-golang.org-x-text",
+      "downloadLocation": "git://golang.org/x/text@v0.0.0-20170915032832-14c0d48ead0c",
```

Download locations for go modules are best pulled through the proxy:
```suggestion
      "downloadLocation": "https://proxy.golang.org/golang.org/x/text/@v/v0.0.0-20170915032832-14c0d48ead0c.zip",
```